packages/api-reference/src/components/Content/ReferenceEndpoint/EndpointDetailsCard.vue

```diff
@@ -60,5 +60,6 @@ const { responses } = useResponses(props.operation)
   font-weight: var(--theme-bold, var(--default-theme-bold));
   font-size: var(--theme-mini, var(--default-theme-mini));
   color: var(--theme-color-2, var(--default-theme-color-2));
+  line-height: 1.33;
```

Comment:
We have variables for this if you want
```suggestion
  line-height: var(--theme-line-height-4, var(--default-theme-line-height-4)); // 18px
```

Reply:
This was just a hacky way to match the height of the headings of the three cards and had nothing to do with line height

<img width="1379" alt="image" src="https://github.com/scalar/scalar/assets/6201407/2c3cc5aa-9357-4ee5-b6ef-9602f5bf7b30">

Not sure how you'd want to match the heights but maybe you can take a look in to this